config/install/media_entity.bundle.twitter.yml

```diff
@@ -0,0 +1,21 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - media_entity_twitter
+id: twitter
+label: Twitter
+description: 'Use this to attach tweets to your content.'
```

Comment:
We should find a better wording for this... Nothing comes to mind, but I would use the name of the service here (Twitter) e.g. "Use this to embed Twitter content on your site."